examples/howto/plot_batch_simulate.py

```diff
@@ -12,6 +12,8 @@
 #          Nick Tolley <nicholas_tolley@brown.edu>
 #          Ryan Thorpe <ryan_thorpe@brown.edu>
 #          Mainak Jas <mjas@mgh.harvard.edu>
+#
+# This project was supported by Google Summer of Code (GSoC) 2024.
```

Comment:
Is this a GSoC requirement?

Reply:
Hi @jasmainak, I received a suggestion from @ntolley  to include a statement in the author section of the example notebook, referencing that the project was supported by GSoC, per their guidelines. 

I'm open to any further suggestions or adjustments you'd like to make.

Reply:
I think this would belong on an `About` page or something? We can chat with Steph to get a full list of funding sources

Reply:
mne-python has a dedicated page: https://github.com/mne-tools/mne-python/blob/3f1c7803f73862dec3e4e85e36e916f2a6d642c9/doc/funding.rst

Reply:
Made an issue #881 !